test/helpers/utils.js

```diff
@@ -30,11 +29,13 @@ export function encodeSigs(sigs = []) {
   return Buffer.concat(sigs.map(s => utils.toBuffer(s)))
 }
 
-export function assertBigNumberEquality(num1, num2, mode='BN') {
-    expect(num1.eq(web3.utils.toBN(num2))).to.be.true;
+export function assertBigNumberEquality(num1, num2) {
+  if (!BN.isBN(num1)) num1 = web3.utils.toBN(num1.toString())
```

Comment:
missed on left `web3.utils.toBN`.
`const BN = web3.utils.toBN' should do ?

Reply:
What missed on left? No for `const BN = web3.utils.toBN', then I'll have to `BN(num)` which is counter intuitive to `new BN(num)`

Reply:
ah! my bad.

Reply:
> missed on left `web3.utils.toBN`.
> `const BN = web3.utils.toBN' should do ?
>

Reply:
> What missed on left? No for `const BN = web3.utils.toBN', then I'll have to `BN(num)` which is counter intuitive to `new BN(num)`